src/dom.js

```diff
@@ -467,3 +467,13 @@ export function openWindowDialog(win, url, target, opt_features) {
   }
   return res;
 }
+
+/**
+ * Whether the element is a script tag with application/json type.
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function isJsonScriptTag(element) {
+  return element.tagName == 'SCRIPT' &&
+            element.getAttribute('type').toUpperCase() == 'APPLICATION/JSON';
```

Comment:
@jridgewell asked to use the normal case of mime type. I agree, it's highly unusual vary the case of mime types. Let's just use lower case.